src/main/resources/org/broadinstitute/hellbender/utils/config/GATKConfig.properties

```diff
@@ -0,0 +1,58 @@
+# ======================================================================================================================
```

Comment:
Is it possible to write a unit test to check that the values in the `GATKConfig.properties` we distribute with the GATK are consistent with the `@DefaultValue` annotations in the `GATKConfig` class?

Reply:
Sure.